pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -283,6 +283,8 @@ def export_testcase(
     onnx_graph, outs = _export(
         model, args, strip_large_tensor_data, large_tensor_threshold,
         input_names=input_names, **kwargs)
+    if isinstance(model, torch.jit.ScriptModule):
+        outs = model(*args)
```

Comment:
https://github.com/pytorch/pytorch/blob/v1.12.0/torch/onnx/utils.py#L717-L718

according to above, the returned `outs` in torch export is None when model is `torch.jit.ScriptModule`.
To clarify that, I think it's better to assert `outs` is None here.

```suggestion
        assert outs is None
        outs = model(*args)
```